src/main/java/com/kainos/ea/backend/repositories/BandRepository.java

```diff
@@ -6,6 +6,5 @@
 import java.util.List;
 
 public interface BandRepository extends CrudRepository<Band, String> {
-
-    List<Band> findAll();
+    List<Band> findAllByOrderByLevelDesc();
```

Comment:
Does findAllOrerByLevelDesc() work? If so please change to that as it would be a clearer name.

Reply:
I've just tried it and it doesn't work. I think JPA just wants it written in this weird way